tests/test_mpi.py

```diff
@@ -21,7 +21,10 @@
 from devito.tools import Bunch
 
 from examples.seismic.acoustic import acoustic_setup
-from tests.test_dse import TestTTI
+try:
+    from tests.test_dse import TestTTI
```

Comment:
why not just `from .test_dse import TestTTI`. If this file is run, test_dse exists there should not be any case where this cannot be imported here

Reply:
Actually just `from test_dse import TestTTI` see e.g.

https://github.com/devitocodes/devito/blob/da2c9a4bc6edd795d7cb046e2ac26de2339e0d8b/tests/test_autotuner.py#L153